### ISSUE TITLE
Shortcode?

### ISSUE THREAD

**Issue by macleodsawyer** [copyblogger/genesis-simple-share#45]
Is there any shortcode option?

**Comment by NicktheGeek** [comment]
@macleodsawyer right now there is not but we have discussed the possibility. It will require some specific changes to the code so that either a specific URL can be included or maybe a post ID. That way it doesn't break everything if it happens to be used outside the loop. Those are features we also have planned.

**Comment by cdils** [comment]
Just chiming in as I'm also working on a solution for a shortcode for GSS in New Rainmaker (for use on a sidebar).

Looks like there's an experiment in play on the sidebar here: http://rainmaker.fm/audio/showrunner/. That may be a hard-coded URL, which isn't ideal, but would be fine at present.

Is this feature still on the books for a future milestone?

**Comment by NicktheGeek** [comment]
@cdils yes, I did did some of the preliminary work that will allow this to work using a defined URL or pull from a custom field in the post. The next step will be to include some additional data which can make for shortcode or template tag based solutions that are pretty well open ended.

**Comment by cdils** [comment]
@NicktheGeek Thanks for the update. Looking forward to those changes.

**Comment by rezzz-dev** [comment]
@NicktheGeek has this been pushed through yet at all?
